pom.xml

```diff
@@ -7,12 +7,10 @@
 
 	<artifactId>opensrp-server-web</artifactId>
 	<packaging>war</packaging>
-
-	<version>1.6.4-SNAPSHOT</version>
-
+	<version>1.6.5-SNAPSHOT</version>
 	<name>opensrp-server-web</name>
 	<description>OpenSRP Server Web Application</description>
-	<url>https://github.com/OpenSRP/opensrp-server-web</url>
+	<url>http://github.com/OpenSRP/opensrp-server-web</url>
```

Comment:
Any reason these urls have been downgraded from the https protocol?

Reply:
I am not sure how this happened. I have upgraded this.